modules/tinymce/src/themes/silver/test/ts/browser/window/SilverDialogPositioningTest.ts

```diff
@@ -0,0 +1,83 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarBody, SugarElement } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import { Dialog } from 'tinymce/core/api/ui/Ui';
+import { WindowParams } from 'tinymce/core/api/WindowManager';
+
+import * as DialogUtils from '../../module/DialogUtils';
+
+describe('browser.tinymce.themes.silver.window.SilverDialogPositioningTest', () => {
+  Arr.each([
+    { label: 'Inline Mode', inline: true },
+    { label: 'Iframe Mode', inline: false }
+  ], (editorMode) => Arr.each([
+    { label: 'LTR Direction', directionality: 'ltr' },
+    { label: 'RTL Direction', directionality: 'rtl' },
+  ], (editorDirectionality) => Arr.each([
+    { label: 'Location Top', location: 'top' },
+    { label: 'Location Bottom', location: 'bottom' }
+  ], (toolbarLocation) => context(`${editorMode.label} ${editorDirectionality.label} Toolbar ${toolbarLocation.label}`, () => {
+    const inlineHook = TinyHooks.bddSetupLight<Editor>({
```

Comment:
Why is this called inlineHook usually the main editor hook is just called hook?

Reply:
Leftover from writing the test within an existing context block. Since it's been moved I'll rename this.